webpack/react_app/components/TargetingHosts/TargetingHostsPage.js

```diff
@@ -39,9 +39,10 @@ const TargetingHostsPage = ({
     <br />
     <TargetingHosts apiStatus={apiStatus} items={items} />
     <Pagination
-      viewType="table"
       itemCount={totalHosts}
-      pagination={pagination}
+      page={pagination.page}
+      perPage={pagination.per_page}
+      perPageOptions={pagination.perPageOptions}
       onChange={args => handlePagination(args)}
       dropdownButtonId="targeting-hosts-pagination-dropdown"
```

Comment:
I am not sure PF4 still support this one,
and I guess the className would be enough?

Reply:
Yeah, we should not use unsupported props and I believe PF4 does not support this, so I'm in favor of dropping this :+1: